modules/keyboardshortcut/keyboardshortcut.js

```diff
@@ -139,6 +139,9 @@ var KeyboardShortcut = {
      * @returns {string} e.key or something close if not supported
      */
     _getKeyboardKey: function (e) {
+        if (e.which === 32) {
```

Comment:
Why do we need a separate check for SPACE? If the  String.fromCharCode(e.which); returns something we don't want to use, so then let's change the condition above to start > 32 and take care of the SPACE in the switch below.